src/main/java/duke/parser/TimeParser.java

```diff
@@ -0,0 +1,42 @@
+package duke.parser;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a TimeParser and consists of methods related to
+ * parsing the string representing the time of tasks.
+ */
+public class TimeParser {
```

Comment:
I like the way that you name it as a noun instead of ParseTime!

Reply:
Thank you!